src/PortfolioPlanning/Components/Plan/DependencyPanel.tsx

```diff
@@ -170,6 +181,18 @@ export class DependencyPanel extends React.Component<IDependencyPanelProps, IDep
         );
     };
 
+    private _showInfoIcon = (item: PortfolioPlanningQueryResultItem, dependsOn: boolean): boolean => {

```

Comment:
You need to take the work item state into account.

Reply:
Right now I have added the comparison of state with "In Progress". I am wondering what is the value of the state here? Is it "in progress" or "active"?